packages/design-system/src/components/Icon/index.js

```diff
@@ -0,0 +1,3 @@
+export { default as Icon } from './Icon';
+
+export { IconSizeName, IconColorName } from './constants';
```

Comment:
```suggestion
export { IconSizeName as IconSize, IconColorName as IconColor } from './constants';
```
can we rename those on export outside?

Reply:
instead of IconColor export IconType